ui/job-view/KeyboardShortcuts.jsx

```diff
@@ -181,20 +181,20 @@ class KeyboardShortcuts extends React.Component {
   };
 
   doKey(ev, callback) {
```

Comment:
Try extracing this function outside of the class or add `eslint-disable-next-line <name of rule>` instead of using `this`.